tests/functional/CSV/Safe/LoaderTest.php

```diff
@@ -92,4 +94,12 @@ public function testLoadCsvWithLogger()
 
         $this->assertFileEquals(__DIR__.'/../data/users.csv', 'vfs://test.csv');
     }
+
+
+    public function pipelineRunner(): PipelineRunnerInterface
+    {
+        return new \Kiboko\Component\Pipeline\PipelineRunner(
+            new NullLogger()
+        );
+    }
```

Comment:
Use a copy of this [PipelineRunner](https://github.com/php-etl/csv-flow/blob/master/tests/functional/PipelineRunner.php)